Content.Server/Roles/Role.cs

```diff
@@ -16,6 +16,12 @@ public abstract class Role
         [ViewVariables]
         public Mind.Mind Mind { get; }
 
+        /// <summary>
+        ///     Faction to which this role is assigned.
+        /// </summary>
+        [ViewVariables]
+        public PlayerFaction? Faction { get; set; }
```

Comment:
На корваксах пишут так:

```suggestion
        public PlayerFaction? Faction { get; set; } // Theta-ShipEvent
```